Framework/PythonInterface/mantid/FunctionWrapper.py

```diff
@@ -0,0 +1,480 @@
+from mantid.simpleapi import FunctionFactory
+
+class FunctionWrapper:
```

Comment:
I see ``class FunctionWrapper(object)`` in the code your provided for the dot operator. May be that's why it works. I'm new to new style classes. https://www.python.org/doc/newstyle/